src/main/app/controller/PostController.ts

```diff
@@ -71,7 +72,9 @@ export class PostController<T extends AnyObject> {
   }
 
   protected getEventName(req: AppRequest<T>): string {
-    if (req.session.isApplicant2) {
+    if (shouldUpdateAos(req)) {
```

Comment:
we could create a custom UPDATE_AOS post-controller that overrides getEventName, and that is triggered by isAosStep urls
doesn't have to block the P2 though

Reply:
Will make another ticket for this.